pkg/deviceshifu/deviceshifumqtt/deviceshifumqtt.go

```diff
@@ -36,6 +38,8 @@ var (
 	MQTTTopic                   string
 	mqttMessageStr              string
 	mqttMessageReceiveTimestamp time.Time
+	mutexBlocking               bool
+	mutexInstructions           map[string]string
```

Comment:
need to use concurrency safe map here?  @BtXin would you give more information?

Reply:
I think this instruction is defined with configmap when the deployment get created. So this may won't have concurrent write scenarios, normal map should suffice. @seanli33056  correct me If I'm wrong.